terraform/calibnet/terraform.tfvars

```diff
@@ -1,15 +1,14 @@
 # This is a set of values to be referenced as variables in the main code
-
 size                  = "s-4vcpu-8gb"
 image                 = "docker-20-04"
 region                = "fra1"
 source_addresses      = ["0.0.0.0/0", "::/0"]
 destination_addresses = ["0.0.0.0/0", "::/0"]
-backups               = "false"
 protocol              = "tcp"
```

Comment:
I don't understand how `protocol` can be anything other than `tcp`.

Reply:
You already have rules for UDP and ICMP. When would we want to open port 22 and 1234 for other protocols than `tcp`?